receiver/googlecloudmonitoringreceiver/config.go

```diff
@@ -11,7 +11,10 @@ import (
 	"go.opentelemetry.io/collector/receiver/scraperhelper"
 )
 
-const minCollectionIntervalSeconds = 60
+const (
+	defaultCollectionInterval = 60 * time.Second // Default value for collection interval
```

Comment:
According to another Googler who works on the monitoring API, we should use a defualt value of 5 minutes for the collection interval.

Reply:
Updated